elm-package.json

```diff
@@ -10,7 +10,7 @@
         "BigInt"
     ],
     "dependencies": {
-        "elm-community/list-extra": "6.1.0 <= v < 7.0.0",
+        "elm-community/list-extra": "7.0.0 <= v < 8.0.0",
```

Comment:
Probably best to widen the constraints instead, so that this can also still be used with older list extra versions. Also worth checking if the dependency is really needed in the first place..

Reply:
Indeed.

Reply:
@zwilias Done.